client/src/components/DatasetInformation/DatasetError.vue

```diff
@@ -126,6 +126,14 @@ function onMissingJobId() {
     errorMessage.value = "No job ID found for this dataset.";
 }
 
+const userEmail = computed<string | null>(() => {
+    if (currentUser && currentUser.value && "email" in currentUser.value) {
```

Comment:
```suggestion
    if (isRegisteredUser(currentUser.value)) {
```
You will need to import isRegisteredUser from `@/api` too.

Reply:
Very nice cleanup - thanks so much!